_posts/2020-05-05-git-it-p1.md

```diff
@@ -7,6 +7,9 @@ bg_url: "https://source.unsplash.com/MSN8TFhJ0is/800x800"
 tags: [home]
 home_title: "Git it! - Simplified Git Tutorial Series"
 home_subtitle: "Learn basic git in < 1 hour"
+toc: true
+h_min: 2
```

Comment:
Everythin working well, remove all h_min and h_max from all the blogs as all the blogs till now should use their default values.